app/assets/stylesheets/members/activities.css

```diff
@@ -208,7 +208,19 @@ div.container.col-md-9 {
   flex-wrap: wrap;
   flex-direction: row;
 }
+.poster-thumbnail .mask {
+    position: absolute;
+    top: 0;
+    height: 100%;
+    width: 140px;
+    opacity: 0;
+    transition: .5s ease;
+    background-color: rgba(62, 53, 53, 0.5);
+}
+.poster-thumbnail:hover .mask {
+    opacity: 1;
```

Comment:
I think that if we also set `cursor: pointer` here the user's cursor will show as a pointy finger, for even more hints that the poster is clickable.

```suggestion
    opacity: 1;
    cursor: pointer;
```